backend/src/v4/routes/maps.js

```diff
@@ -546,39 +494,26 @@ function requestMapTile(req, res, domain, uri) {
 	});
 }
 
-function requestHereMapTile(req, res, domain, uri) {
-	uri += "?apiKey=" + config.here.apiKey;
+function requestHereMapTile(req, res, domain, resource, style, features) {
+	const featureList = features ? features.split(",") : [];
```

Comment:
I'm a bit confused about how this works. I can't see anywhere that you pass in a comma separated feature list? or am I missing something? 

If you are, why? you may as well just pass it in as an array to begin with?

Reply:
This was in case a comma separated feature list was supplied via query string, which doesn't happen, and we wanted to merge it with other features set internally.

Removed.